blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionServer.java

```diff
@@ -219,4 +127,74 @@ public void exceptionCaught(ChannelHandlerContext ctx, ExceptionEvent e) throws
             return pipeline;
         }
     }
+    static class Processor {
```

Comment:
It made sense to me as a subclass in the same namespace, and it didn't need a reference to an instance of the parent class.  But I have no problem moving it out, if you think it isn't clear.